app/controllers/admins/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class Admins::ReviewsController < ApplicationController
+  before_filter :authorized?
+
+  def index
+    @items = Item.all
+  end
+
+  def destroy
+    @review = Review.find(params[:id])
+    @review.destroy
+    redirect_to admins_reviews_path
+  end
+
+  private
+
+  def authorized?
```

Comment:
Do the same thing you did to the `Admins::UsersController` - you can inherit from `AdminsController` and get rid of `before_filter :authorized?` as well as the private method here